app/src/androidTest/java/com/example/bigowlapp/activity/InvitationConfirmationActivityTest.java

```diff
@@ -0,0 +1,48 @@
+package com.example.bigowlapp.activity;
+
+import android.app.Activity;
+import android.app.Instrumentation;
+
+import androidx.test.rule.ActivityTestRule;
+
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.action.ViewActions.click;
+import static androidx.test.espresso.matcher.ViewMatchers.withId;
+import static androidx.test.platform.app.InstrumentationRegistry.getInstrumentation;
+import static org.junit.Assert.*;
+import com.example.bigowlapp.R;
+
+public class InvitationConfirmationActivityTest {
+
+    @Rule
+    public ActivityTestRule<InvitationConfirmationActivity> invitationConfirmationActivityTestRule = new ActivityTestRule<InvitationConfirmationActivity>(InvitationConfirmationActivity.class);
+    private InvitationConfirmationActivity invitationConfirmActivity =null;
+    Instrumentation.ActivityMonitor monitor = getInstrumentation().addMonitor(HomePageActivity.class.getName(), null, false);
+
+    @Before
+    public void setUp() throws Exception {
+
+        invitationConfirmActivity = invitationConfirmationActivityTestRule.getActivity();
+    }
+
+    @Test
+    public void launchHomePageActivityOnButtonClick(){
+        assertNotNull(invitationConfirmActivity.findViewById(R.id.returnHome));
```

Comment:
Since an important aspect of this page is the message, maybe you should assert that it is visible as a part of this UI test.

Reply:
The message is not part of this page. I have another PR for testing that will include the message and I will be able to assert the message is visible

Reply:
I'm talking about the text on the page:
![image](https://user-images.githubusercontent.com/31963426/98332336-7c0dca00-1fcc-11eb-88c1-9c835ceff720.png)

Reply:
ok, I see. I thought you were talking about the sent message. I can add that now